Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM httpd:2.4 AS base
+
+# Set version label
+LABEL maintainer="lycheeorg"
+
+# Environment variables
+ENV PUID='1000'
+ENV PGID='1000'
+ENV USER='lychee'
+ENV PHP_TZ=UTC
+
+# Multi-stage build: Build static assets
+# This allows us to not include Node within the final container
+FROM python:3.14-rc-bookworm AS python_builder
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+COPY template ./template
+COPY utils ./utils
+COPY docs ./docs
+COPY gen.py .
```

Comment:
IIRC this will lead to lots of image layers. Then again, we're discarding that anyway, and this may lead to fewer rebuilds when working locally, so I guess we may as well go with this.